test/benchmarks/taylor_pi.inputs

```diff
@@ -0,0 +1,14 @@
+pi_1000000_runs
+taylor_pi
+i:I
+1000000
+
+31415916535897744
+
+pi_3000000_runs
```

Comment:
Please keep single execution case for a start. Something that runs in milliseconds range.

Reply:
This is on my machine:
```
fizzy/execute/taylor_pi/pi_1000000_runs      52600 us        51521 us           14
 wabt/execute/taylor_pi/pi_1000000_runs      80834 us        79290 us            9
wasm3/execute/taylor_pi/pi_1000000_runs      12373 us        12234 us           58
fizzy/execute/taylor_pi/pi_3000000_runs     152631 us       150737 us            4
 wabt/execute/taylor_pi/pi_3000000_runs     235717 us       233095 us            3
wasm3/execute/taylor_pi/pi_3000000_runs      36830 us        36516 us           19
```

What is your upper bound?

Reply:
Leave the first one. The second runs over 2 seconds with sanitizers on CI.